constants/chatbotCommands.py

```diff
@@ -1761,98 +1791,105 @@ async def mpMake() -> Optional[str]:
             host_user_id=-1,
             is_tourney=True,
         )
-        multiplayer_match = await matchList.getMatchFromChannel(chan)
-        assert multiplayer_match is not None
 
-        await match.sendUpdates(multiplayer_match["match_id"])
-        return f"Tourney match #{matchID} created!"
-
-    # def mpJoin() -> str:
-    #     if len(message) < 2 or not message[1].isnumeric():
-    #         raise exceptions.invalidArgumentsException(
-    #             "Incorrect syntax: !mp join <id>",
-    #         )
-    #     matchID = int(message[1])
-    #     userToken = await  tokenList.getTokenFromUsername(fro, ignoreIRC=True)
-    #     if userToken is None:
-    #         raise exceptions.invalidArgumentsException(
-    #             f"No game clients found for {fro}, can't join the match. "
-    #             "If you're a referee and you want to join the chat "
-    #             f"channel from IRC, use /join #multi_{matchID} instead.",
-    #         )
-
-    #     await osuToken.joinMatch(userToken["token_id"], matchID)
-    #     return f"Attempting to join match #{matchID}!"
-
-    async def mpClose() -> Optional[str]:
-        if not (userID := await userUtils.getIDSafe(fro)):
-            raise exceptions.userNotFoundException("No such user")
+        async with redisLock(f"{match.make_key(multiplayer_match['match_id'])}:lock"):
```

Comment:
i'm not necessarily challenging your choice, i just want to make sure we're aligned - what did you add the lock for here?

Reply:
good catch, i don't think it's needed there i think i just rush added it

Reply:
resolved in 2e0740d